### ISSUE TITLE
add search in the add product page?

### ISSUE THREAD

**Issue by lankeeyankee** [lat9/edit_orders#92]
It would be great to be able to have a auto-suggest type search feature when adding an item. We use edit orders throughout the day, every day, and we are pretty familiar with model numbers and product titles. Having a search/suggestion box would greatly speed up entering items when there are many thousands of products and hundreds of categories.

**Comment by lat9** [comment]
I was wondering where you'd disappeared to!  I'm getting close to finishing another integration, but this is quite doable ... something akin to the `ZX AJAX Product Search plugin`?

**Comment by lankeeyankee** [comment]
Yes, something akin to that plugin would be great, or even just a field to enter the model number to then go to the next step.

**Comment by lat9** [comment]
Circling back on this.  The "Add Product" processing already has a product-search feature:
![image](https://user-images.githubusercontent.com/2685585/110122073-2801cf00-7d8d-11eb-9be9-6bb065a76b0c.png)

Does that satisfy your requirement?

**Comment by lankeeyankee** [comment]
Yes, that is exactly what I was asking for. The only suggestion I have after using it for a while is weighting the results a little better, such as sort by results in model, then title, then description. I think that would help make the top results more relevant when searching by keyword instead of a model number.

**Comment by lat9** [comment]
Thanks for the update and the details.  I'll see if I can get that feature into the next EO version.
